Makefile

```diff
@@ -34,18 +34,14 @@ fmt:
 .PHONY: fmt
 
 # Run all the linters
-lint: bin/golangci-lint bin/misspell
```

Comment:
Do we need to remove `misspell`?

Reply:
My third bullet in the PR summary briefly explains this. I thought misspell is [already installed](https://golangci-lint.run/usage/linters) and used by out `golangci-lint' step by enabling it in the [golangci.yml file](https://github.com/stripe/stripe-cli/blob/master/.golangci.yml#L23)?

Reply:
following up - I was able to provoke mispell to surface an error locally by running `make lint` even with the mispell install step removed.